### DISCUSSION TITLE
Add MLJAR AutoML as a task

### DISCUSSION THREAD

**Discussion by pplonski** [PrefectHQ/prefect#4467] [Ideas]
Hey there! Just wondering, is it possible to add MLJAR AutoML as a task? MLJAR AutoML is an open-source package for working with tabular data. Link to repo -> https://github.com/mljar/mljar-supervised I can try to create PR

**Comment by zanieb** [comment]
Yeah! This looks like it'd work great with our Artifacts API for publishing those graphs / tables.
